techniques/systemSettings/userManagement/userManagement/7.0/userManagement.st

```diff
@@ -215,6 +228,30 @@ bundle agent check_usergroup_user_parameters
         classes       => classes_generic("usermanagement_user_password_${usergroup_user_index}"),
         ifvarclass    => "!usermanagement_user_pwempty_${usergroup_user_index}.usermanagement_user_exists_${usergroup_user_index}.usermanagement_user_pweverytime_${usergroup_user_index}.usermanagement_user_checkpres_${usergroup_user_index}";
 
+    aix::
+      "/etc/security/passwd"
+        create        => "false",
+        edit_line     => ncf_ensure_AIX_password("${usergroup_user_login[${usergroup_user_index}]}", "${usergroup_user_password_aix[${usergroup_user_index}]}"),
+        edit_defaults => noempty_backup,
+        classes       => kept_if_else("usermanagement_user_password_${usergroup_user_index}_kept", "usermanagement_user_password_${usergroup_user_index}_repaired", "usermanagement_user_password_${usergroup_user_index}_error"),
+        ifvarclass    => "(usermanagement_login_add_${usergroup_user_index}_repaired.usermanagement_user_pwoneshot_${usergroup_user_index}.!usermanagement_user_pwempty_${usergroup_user_index})|(usermanagement_user_update_${usergroup_user_index}.usermanagement_user_exists_${usergroup_user_index}.usermanagement_user_pweverytime_${usergroup_user_index}.!usermanagement_user_pwempty_${usergroup_user_index})";
```

Comment:
so we can't create a class based on that, as it will delay usermanagement_login_add_${usergroup_user_index}_repaired